.github/workflows/biome.yml

```diff
@@ -0,0 +1,43 @@
+name: "biome"
+
+on:
+  workflow_call:
+    inputs:
+      head_sha:
+        required: true
+        type: string
+
+env:
+  NODE: "18"
+  FRONTEND_MONOREPO_DIR: "web"
+
+jobs:
+  main:
+    name: "biomejs"
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: hmarr/debug-action@v3.0.0
+
+      - name: Configure git
+        shell: bash
+        run: |
+          set -xeuo pipefail
+          git config --global user.name 'robot-ci-heartex'
+          git config --global user.email 'robot-ci-heartex@users.noreply.github.com'
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.ref }}
```

Comment:
inputs.head_sha

Reply:
Should we review this in other workflows?